crates/vm2/src/decommit.rs

```diff
@@ -98,7 +98,8 @@ impl WorldDiff {
         let code = world.decommit_code(code_hash);
         if is_new {
             // Decommitter can process two words per cycle
```

Comment:
Move this comment one line down.

Reply:
Also, I wonder if there is a clearer way of writing "divide by 64, rounding up"

Reply:
There is [`div_ceil`](https://doc.rust-lang.org/core/primitive.u64.html#method.div_ceil).

Reply:
div_ceil looks perfect. Communicates the intention well, so I'd replace (x + 63) / 64 and (x + 1) / 2 with this method.

Reply:
Yeah, I did just that 😄